File: URI/Java/src/URI/_1019.java
package URI;

import java.util.Scanner;

public class _1019 {
    
    public static void main(String args[])
    {
        Scanner sc = new Scanner(System.in);
        int segundos = sc.nextInt();
        int horas = segundos / 3600;
        segundos = segundos % 3600;
        int minutos = segundos / 60;
        segundos = segundos % 60;
        
        System.out.println(horas + ":" + minutos + ":" + segundos);
    }
}

File: URI/1019.cpp
#include<iostream>
using namespace std;

int main(){
	int time, horas, minutos;

	cin >> time;

	horas = time / 3600;
	time -= horas * 3600;

	minutos = time / 60;
	time -= minutos * 60;

	cout << horas << ":" << minutos << ":" << time << endl;

	return 0;
}